.github/CODEOWNERS

```diff
@@ -300,6 +300,7 @@
 
 
 # Navigation bar project.
+/core/templates/domain/bottom_navbar/ @nithusha21 @showtim3
```

Comment:
I don't think `core/templates/domain/bottom_navbar/bottom-navbar-status.service.ts` (Service) should be included in the domain directory, can you move it to the services dir?

Reply:
The sidebar service is also present in domain, so i followed that convention.

Reply:
hmmm that's not expected, thanks for raising it can you create an issue to move that file to the service layer and assign it to me? I can resolve it in a separate PR!

Reply:
https://github.com/oppia/oppia/issues/9726